server/src/weaverbird/pipeline/steps/ifthenelse.py

```diff
@@ -22,13 +22,13 @@ class Config(PopulatedWithFieldnames):
 
 
 class IfthenelseStep(BaseStep, IfThenElse):
-    class Config(PopulatedWithFieldnames):
-        ...
-
     name: Literal["ifthenelse"] = "ifthenelse"
     new_column: ColumnName = Field(alias="newColumn")
 
+    class Config(BaseConfig):
+        allow_population_by_field_name = True
```

Comment:
Is this needed ? It's in boith `BaseStep` and `IfthenElse`. Or does the multiple inheritance F up pydantic's config ?

Reply:
I know but it fucks up mypy ;)